.travis.yml

```diff
@@ -67,7 +67,24 @@ script:
   - java -version
   - if [ -n "$CLANG_FORMAT_PATH" ]; then bin/clangfmt --test; fi
   - bin/scalafmt --test
-  - bash sbt -J-Xmx2G rebuild "set scriptedBufferLog in sbtScalaNative := false" test-all
+  - sbt -J-Xmx2G rebuild
+  - mkdir utest
+  - pushd utest
+  - git init .
+  - git remote add -t cross-native Duhemm https://github.com/Duhemm/utest.git
```

Comment:
Btw, sbt-crossproject is now at `0.2.0`, we released newer version but haven't announced it yet.